definitions/npm/amqplib_v0.5.x/flow_v0.60.1-/amqplib_v0.5.x.js

```diff
@@ -0,0 +1,286 @@
+// @flow
+
+declare module 'amqplib' {
+    declare export type AMQPError = any;
```

Comment:
Indentation should be 2 spaces IIRC. Did prettier not run on your machine? It should've been autoformatted.